eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FieldValue/Handler.php

```diff
@@ -147,4 +157,50 @@ protected function lowerCase(string $string): string
     {
         return $this->transformationProcessor->transformByGroup($string, 'lowercase');
     }
+
+    /**
+     * @param mixed $value
+     *
+     * @return mixed
```

Comment:
```suggestion
     * @return array<string>|string
```

Reply:
It may return any type or array of any types.

Reply:
Then I'd suggest marking this as
```suggestion
     * @return scalar|array<scalar>
```
so that static analysis is not disabled completely.

Do note that for arrays you're always binding a `Connection::PARAM_STR_ARRAY`.